package.json

```diff
@@ -104,4 +102,4 @@
     "vercel": "^21.2.3",
     "whatwg-fetch": "^3.5.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
is this a prettier thing? Is it possible to remove whitespace?

Reply:
it is a prettier thing, but in general, all files should end with a trailing whitespace.